### ISSUE TITLE
Version 24 has issues, how to upgrade to version 23?

### ISSUE THREAD

**Issue by nakoshi-satamoto** [ElementsProject/lightning#7361]
See issue https://github.com/ElementsProject/lightning/issues/7180

Ever since going to version 24 and sub versions after 24, I have had issues with having successful outbound payments. The log file also gets filled with gossip related errors. Is it possible to safely go back to version 23?

**Comment by grubles** [comment]
Is that while using the `pay` plugin or `renepay`? I don't think you can move back to 23.x since moving to 24.x would have likely upgraded your database and there is no support for downgrading.

**Comment by nakoshi-satamoto** [comment]
On 2024-06-01 18:46, grubles wrote:
> Is that while using the `pay` plugin or `renepay`? I don't think you
> can move back to 23.x since moving to 24.x would have likely upgraded
> your database and there is no support for downgrading.

I am using the pay command. What is renepay? Which is the proper command 
to use for payments?

**Comment by crc32** [comment]
I cannot pay using Ride The Lightning, `pay` or `renepay`.

`renepay` results in:
`{
   "code": 205,
   "message": "Destination is unknown in the network gossip."
}`

This is paying to strike, which I can pay just fine to from LND.

**Comment by chrisguida** [comment]
No, there is no way to downgrade your database to a previous version. If you want a previous version, spin up a new node.

**Comment by vincenzopalazzo** [comment]
>If you want a previous version, spin up a new node.

Unfortunatly this is bad, but this is the only way that you have to run a previous version. However, this is a tech solution not a real one because you can not bring your channels with you.

**Comment by whitslack** [comment]
There aren't *that* many database schema changes between releases of CLN. If you were desperate, you could manually reverse the changes in your database.

**Comment by rustyrussell** [comment]
AI cleanup of old issues.  If wrong, please open a new issue.


Triage decision summary:
User reports payment and gossip issues after upgrading to v24 in June 2024. Issue references #7180 but provides no resolution details. Nearly 8 months have passed with no updates; likely resolved in subsequent patch releases or user found workaround.
